model/model.wl

```diff
@@ -590,10 +594,12 @@ evaluateState[state_]:= Module[{distance,sol,params,longData,thisStateData,model
 	(* if we cannot get smooth enough then use Nelder-Mead Post-processing \[Rule] false *)
 	
 	fitParams=Exp[#]&/@KeyMap[ToString[#]&, Association[fit["BestFitParameters"]]];
+	(* TODO: try using VarianceEstimatorFunction\[Rule](1)& *)
 	standardErrors=Exp[#]&/@KeyMap[ToString[#]&, AssociationThread[{r0natural,importtime},
 	     fit["ParameterErrors",
 	     ConfidenceLevel->0.97]]];
-	     
+	gofMetrics=goodnessOfFitMetrics[fit["Residuals"],longData];
```

Comment:
can we compute this above and then dump to CSV before we return? 

Something like `Export["model/test/fit.csv", goodnesOfFitMetrics[...]]`

Reply:
The gof metrics are per state, so if we want to have a CSV (and ultimately plots) of all these things, we'll either have to refactor a bit to separate out the simulation section or duplicate everything that goes before the simulation and run that separately.

Does that make sense?

Reply:
My PR refactors the export calls per state!